packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -57,51 +50,32 @@ const OdysseyProviderInner = <SupportedLanguages extends string>({
   stylisPlugins,
   themeOverride,
   translationOverrides,
-}: OdysseyProviderProps<SupportedLanguages>) => {
-  const { contrastMode } = useContrastContext();
-
-  const memoizedThemeProps = useMemo(
-    () => ({
-      ...themeOverride,
-      odysseyContrastMode: contrastMode,
-    }),
-    [themeOverride, contrastMode],
-  );
-
-  return (
-    <OdysseyCacheProvider
-      emotionRoot={emotionRoot}
-      emotionRootElement={emotionRootElement}
-      hasShadowDom={Boolean(shadowRootElement || shadowDomElement)}
-      nonce={nonce}
-      stylisPlugins={stylisPlugins}
-    >
-      <OdysseyThemeProvider
-        designTokensOverride={designTokensOverride}
-        shadowDomElement={shadowDomElement}
-        shadowRootElement={shadowRootElement}
-        themeOverride={memoizedThemeProps}
-      >
-        <ScopedCssBaseline sx={scopedCssBaselineStyles}>
-          <OdysseyTranslationProvider<SupportedLanguages>
-            languageCode={languageCode}
-            translationOverrides={translationOverrides}
-          >
-            {children}
-          </OdysseyTranslationProvider>
-        </ScopedCssBaseline>
-      </OdysseyThemeProvider>
-    </OdysseyCacheProvider>
-  );
-};
-
-const OdysseyProvider = <SupportedLanguages extends string>({
   contrastMode,
-  ...providerProps
 }: OdysseyProviderProps<SupportedLanguages>) => (
-  <ContrastModeProvider contrastMode={contrastMode}>
-    <OdysseyProviderInner {...providerProps} />
-  </ContrastModeProvider>
+  <OdysseyCacheProvider
+    emotionRoot={emotionRoot}
+    emotionRootElement={emotionRootElement}
+    hasShadowDom={Boolean(shadowRootElement || shadowDomElement)}
+    nonce={nonce}
+    stylisPlugins={stylisPlugins}
+  >
+    <OdysseyThemeProvider
+      designTokensOverride={designTokensOverride}
+      shadowDomElement={shadowDomElement}
+      shadowRootElement={shadowRootElement}
+      themeOverride={themeOverride}
+      contrastMode={contrastMode}
```

Comment:
I'm concerned about exposing this `contrastMode` prop. I get why it's here, but we shouldn't need it. For now, it's a fail-safe.

I'm just hoping it won't become a liability. Ideally, we'd mark it as `@experimental` in TypeScript.